### PR TITLE
Fixes #18118: Remove syslog configuration from system techniques

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/18118

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

File: techniques/system/distributePolicy/1.0/metadata.xml [hunk 1]
```diff
@@ -50,15 +38,12 @@
 
   <SYSTEMVARS>
     <NAME>CLIENTSFOLDERS</NAME>
-    <NAME>SYSLOGPORT</NAME>
     <NAME>SHARED_FILES_FOLDER</NAME>
-    <NAME>STORE_ALL_CENTRALIZED_LOGS_IN_FILE</NAME>
```
Comment (ncharles) [inline techniques/system/distributePolicy/1.0/metadata.xml:55]: ha, this one is a regression - we should be able to use this value to store all log in all.log
maybe a configuration in rudder-relayd ?

File: techniques/system/common/1.0/check-rsyslog-version [hunk 2]
```diff
@@ -1,31 +0,0 @@
-#!/bin/sh
```
Comment (ncharles) [inline techniques/system/common/1.0/check-rsyslog-version:1]: it should also be removed from the metadata.xml https://github.com/Normation/rudder-techniques/pull/1617/files#diff-3d9c01bf53ab8104c0bc76244daad2f3R78

File: techniques/system/common/1.0/promises.st [hunk 3]
```diff
@@ -280,227 +277,15 @@ bundle agent initialize_ncf()
...
-#######################################################
-# Check the log system, and configure it accordingly
-#
-# Note: if any configurations are changed here, corresponding services must be
-# restarted in the restart_services bundle
-
-bundle agent check_log_system
-{
-  vars:
-    debian::
```
Comment (ncharles) [inline techniques/system/common/1.0/promises.st:328]: these variables are used in remove_rudder_syslog_configuration  https://github.com/Normation/rudder-techniques/pull/1617/files#diff-75507fb0a9433bdb509596c7d5251f35L525

**Review by ncharles — CHANGES_REQUESTED** [review]
this is awesome to remove so many deprecation
some minor change requested, fot the required variables to delete rsyslog content, and removing ref to check_rsyslog_version in metadata

File: techniques/system/common/1.0/promises.st (hunk 3 above)
Comment (amousset) [reply techniques/system/common/1.0/promises.st:328]: good spot!

**Comment by amousset** [comment]
PR updated with a new commit

**Review by ncharles — APPROVED** [review]

**Comment by ncharles** [comment]
OK, squash merging this PR

**Pull request merged**